QuotationsWebApi/Dtos/QuotationDtos/QuotationForCreationDto.cs

```diff
@@ -0,0 +1,21 @@
+﻿using QuotationsWebApi.Entities;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace QuotationsWebApi.Dtos
+{
+    public class QuotationForCreationDto
+    {
+        [Required(ErrorMessage = "Name is required")]
+        [StringLength(20, ErrorMessage = "Name can't be longer than 20 characters")]
+        public string Name { get; set; }
+
+        [Range(0.1, int.MaxValue, ErrorMessage = "Please enter a value bigger than {0.1}")]
+        public float Price { get; set; }
+        [Required(ErrorMessage = "Id of a dossier is required")]
+        public Guid DossierId { get; set; }
+        public Status Status { get; set; }
+        public DateTime DataCreated { get; set; }
+        public DateTime DataUntilValid { get; set; }
```

Comment:
in the database configuration, you set all fields as required, they should be required here also